docker/jenkins/publish-build.sh

```diff
@@ -136,7 +144,9 @@ timestamp=$(date +"%Y-%m-%dT%H:%M:%S%z")
 
 # Determine release channel (build type) and flower
 RSTUDIO_ROOT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && cd ../.. && pwd )"
-channel="$(cat "$RSTUDIO_ROOT_DIR/version/BUILDTYPE" | tr '[ ]' '-' | tr -d '[:space:]')"
+if [ -z "$channel" ]; then
+    channel="$(cat "$RSTUDIO_ROOT_DIR/version/BUILDTYPE" | tr '[ ]' '-' | tr -d '[:space:]')"
```

Comment:
The `hourly` build doesn't actually update the `BUILDTYPE` file, so assuming this might be incorrect. Technically it would work with the logic we have here, but it might be better to just make `channel` a required parameter? See [this pr](https://github.com/rstudio/rstudio/pull/12615) for those changes.

Reply:
The intent is to continue using the `BUILDTYPE` for Daily, Preview, and Release channels as we have been. But to allow us to override it with an Hourly channel when we need to

Reply:
With a clear help message that says we will use the `BUILDTYPE` file if `channel` is not set, I think this is fine